warehouse-drone/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.5.1-slim
+
+LABEL name="warehouse-ruby"
+LABEL version="2.5.1"
+
+ENV LANG     C.UTF-8
+ENV LANGUAGE C.UTF-8
+ENV LC_ALL   C.UTF-8
+
+RUN apt-get update -qq && apt-get install -y \
```

Comment:
Do you need this step? This is added for the other image to use `cURL` against an https URL and to import a GPG key. I suspect you don't.

Reply:
Ah, good spot. Checked the warehouse build in drone without these and all seems good.